CMakeLists.txt

```diff
@@ -105,6 +105,11 @@ cmake_dependent_option(BUILD_SHARED_LIBS
 )
 mark_as_advanced(BUILD_SHARED_LIBS)
 
+# On Mac OS X, set the dir. included as part of the installed library's path:
+if (BUILD_SHARED_LIBS AND NOT DEFINED CMAKE_INSTALL_NAME_DIR)
+  set(CMAKE_INSTALL_NAME_DIR "${CMAKE_INSTALL_PREFIX}/lib")
```

Comment:
Instead of `/lib`, use `${CMAKE_INSTALL_LIBDIR}` and move this block up to ln 27 to be with the other install dir stuff

Reply:
Done.